components/Product/models.rst

```diff
@@ -1,71 +1,199 @@
 Models
 ======
 
-The Product
------------
+.. _component_product_model_product:
+
+Product
+-------
+
+The **Product** model represents every unique product in the catalog.
+By default it contains the following properties:
+
++-------------+---------------------------------------+
+| Property    | Description                           |
++=============+=======================================+
+| id          | Unique id of the product              |
++-------------+---------------------------------------+
+| archetype   | An archetype assigned to this product |
++-------------+---------------------------------------+
+| availableOn | When the product is available         |
++-------------+---------------------------------------+
+| attributes  | Attributes assigned to this product   |
++-------------+---------------------------------------+
+| variants    | Variants assigned to this product     |
++-------------+---------------------------------------+
+| options     | Options assigned to this product      |
++-------------+---------------------------------------+
+| createdAt   | Product's date of creation            |
++-------------+---------------------------------------+
+| updatedAt   | Product's date of update              |
++-------------+---------------------------------------+
+| deletedAt   | Product's date of deletion            |
++-------------+---------------------------------------+
```

Comment:
The concept was to do it like that as far as i remember: https://github.com/Sylius/Sylius-Docs/pull/346/files#diff-c85f3866221a0516d7d9ba284d4a4940R59